code/modules/language/vox.dm

```diff
@@ -1,7 +1,7 @@
 /datum/language/vox
 	name = "Vox-pidgin"
 	desc = "The common tongue of the various Vox ships making up the Shoal. It sounds like chaotic shrieking to everyone else."
-	key = "V"
+	key = "v"
```

Comment:
This can't be `v` unfortunately, since that conflicts with 'Voltaic' (the Ethereal language):
https://github.com/pariahstation/Pariah-Station/blob/88ab81e75b607cca8118d4b064b88039d6d994df/code/modules/language/voltaic.dm#L2-L5
Currently when you try to speak Vox-pidgin ingame it checks `can_speak_language(/datum/language/voltaic)` and fails, since Vox aren't able to speak that.
https://github.com/pariahstation/Pariah-Station/blob/88ab81e75b607cca8118d4b064b88039d6d994df/code/modules/mob/mob_say.dm#L166-L172
Presumably this is why it was set to a capital V in the first place.

Reply:
ah, of course that makes sense, figures.